raco/backends/myria/myria.py

```diff
@@ -2003,14 +2017,20 @@ def fire(self, op):
             else:
                 new_statements.append(MyriaSink(idb_controller))
         op.args = new_statements
-
-        op.has_been_transformed = True
         return op
 
     def __str__(self):
         return ("DoUntilConvergence")
 
 
+def idb_until_convergence(**kwargs):
+    ret = [DoUntilConvergence(), RemoveEmptyFilter(), StoreFromIDB(),
+           RemoveSingleSplit()]
+    if kwargs.get('async_ft', 'None') is not None:
```

Comment:
Why the second argument? The default of one-arg `get()` is `None`, of course. You could really just say `kwargs.get('async_ft')`, since apparently any non-null value is equivalent.